services/order.py

```diff
@@ -0,0 +1,30 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None) -> Order:
+    with transaction.atomic():
+        order = Order.objects.create(user=User.objects.get(username=username))
```

Comment:
Ensure that the `User` object retrieval with `User.objects.get(username=username)` handles potential exceptions, such as `User.DoesNotExist`, to avoid runtime errors if the username is not found.